src/db_impl.cc

```diff
@@ -326,15 +326,26 @@ Status TitanDBImpl::OpenImpl(const std::vector<TitanCFDescriptor>& descs,
                           descs[i].options.table_factory /*base_table_factory*/,
                           titan_table_factories[i]}));
     column_families[(*handles)[i]->GetID()] = descs[i].options;
-    if (!descs[i].options.disable_auto_compactions) {
-      cf_with_compaction.push_back((*handles)[i]);
-    }
   }
-  TEST_SYNC_POINT_CALLBACK("TitanDBImpl::OpenImpl:BeforeOpenBlobFileSet", this);
   s = blob_file_set_->Open(column_families);
   if (!s.ok()) {
     return s;
   }
+  for (size_t i = 0; i < handles->size(); i++) {
+    auto rocks_cf_handle =
+        static_cast_with_check<rocksdb::ColumnFamilyHandleImpl>((*handles)[i]);
+    auto titan_handle = new TitanColumnFamilyHandle(
+        *rocks_cf_handle,
+        blob_file_set_->GetBlobStorage(rocks_cf_handle->GetID()).lock());
+    (*handles)[i] = titan_handle;
+    delete rocks_cf_handle;
```

Comment:
Question: is this rocks_cf_handle be used by TitanColumnFamilyHandle anymore? As it's used as the const ColumnFamilyHandleImpl& in TitanColumnFamilyHandle()

Reply:
OK. Saw the other rocksdb PR. It's Ref() ed.